speakers.js

```diff
@@ -0,0 +1,67 @@
+const speakers = [
+  {
+    name: 'Viviana Angulo', position: 'Co-Founder Pista8', description: 'Accelerator of international and business collaborative format.', photo: 'https://drive.google.com/uc?export=download&id=1B8lJulbXbs_Py6CZ28wHOc7ZEa9quvkd',
+  },
+
+  {
+    name: 'Sebastian Ibáñez', position: 'Consultor PYMES', description: 'With a background in Management Engineering and a Master in Innovation and Entrepreneurship Management.', photo: 'https://drive.google.com/uc?export=download&id=1KDVHabPOpgHGwLPt-y7YKX6yirXtHLeZ',
+  },
+
+  {
+    name: 'Pedro Rey Puma', position: 'Founder UTN blockchain Lab', description: 'Marketing Manager for the global exchange okx.com, professor in the cryptoeconomics course at the Universidad del Cuyo.', photo: 'https://drive.google.com/uc?export=download&id=184r7KWTFrK46gQHCaNb3NptX9vYyTQRI',
+  },
+
+  {
+    name: 'Yamila Peralta', position: 'CEO Women Entrepreneurs', description: 'PhD in Psychology (University of Barcelona, ​​Spain). Master in Work Psychology of Organizations and Human Resources (Univ. de Barcelona, ​​Spain).', photo: 'https://drive.google.com/uc?export=download&id=1df1QQTtIop23rhHKtBRiD5f8_vRQq9VE',
+  },
+
+  {
+    name: 'Marcelo Duran', position: 'Editor at Punto BO magazine', description: 'Degree in Communication, with a mention in advertising from the UCB, Master of Fine Arts in Computer Arts from the Tracor Institute. Quantum Motors', photo: 'https://drive.google.com/uc?export=download&id=1ZE16c7hV9wrXJd4ZK8Z0tdIBsiKe2meC',
+  },
```

Comment:
- [ ] This section contains six speakers but you have only 5, having a nice design that sticks to the original increases your chances for a job as recruiters will be checking your GitHub projects 👍 👍
- [ ] Also these section articles are organized in two columns of three articles each.

Your design

<img width="628" alt="Screen Shot 2022-10-21 at 00 18 47" src="https://user-images.githubusercontent.com/74501523/197070002-dfd0e282-babf-4bab-b8fa-32c2c7652c03.png">

Original design

<img width="971" alt="Screen Shot 2022-10-21 at 00 19 14" src="https://user-images.githubusercontent.com/74501523/197070040-c63be68b-5f38-46ed-998e-ce7b36f1966f.png">